### PR TITLE
User manual and hyde template

### PR DESCRIPTION
**Pull request by bchapuis**
The idea is to create a more complete user manual and to improve the navigation with a secondary menu that point to html anchors.

### CODE REVIEW

**Comment by fd0** [comment]
This looks really good, thanks a lot! I like the layout.

The only thing that I don't like (but that's really minor) is the homepag's title, with the leading middot.

Good work!

File: user-manual.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+scanned 764 directories, 1816 files in 0:00
+[0:00] 100.00%  0B/s  1.582 GiB / 1.582 GiB  2580 / 2580 items  0 errors  ETA 0:00
+duration: 0:00, 6572.38MiB/s
+snapshot 79766175 saved
+{% endhighlight %}
+
+You can even backup individual files in the same repository. 
+
+{% highlight console %}
+$restic -r /tmp/backup backup ~/work.txt
```
Comment (fd0) [inline user-manual.md:105]: Add a space between the `$` and `restic`, the same goes for all the other sample command lines.

**Comment by fd0** [comment]
I've fixed a few minor things (including the spacing), is it okay for you if I just merge your PR and put my changes on top? I'd like to deploy this as soon as possible :+1:

**Comment by bchapuis** [comment]
No problem for the merge, I have no problem with polishing. :)

The same for the future PR, if you have something in mind for finalizing do
not hesitate to go forward.

Regards,

Bertil

**Comment by fd0** [comment]
Awesome! And again: very good work! :+1:

**Pull request merged**